docs/index-server.md

```diff
@@ -119,6 +119,16 @@ be written to file.
 The user can set the location for these file by using 'carbon.indexserver.temp.path'. By default
 table path would be used to write the files.
 
+## Prepriming
+The caching of the datamaps in the Index Server will start once the query is fired on the table for
+the first time. All the datamaps will be loaded if a count(*) query is fired and only the required
```

Comment:
No need to mention count(*). cache can be loaded by any query

Reply:
done